ews - get-items enhancement (#27043)
demisto/content: Packs/MicrosoftExchangeOnPremise/ReleaseNotes/1_0_12.md

#### Integrations

##### EWS v2

> - Added an entry for *categories* in the **ews-get-items** and **ews-get-items-from-folder** commands.

Reviewer: ```suggestion
- Added the *categories* field as an output of the ***ews-get-items*** and ***ews-get-items-from-folder*** commands.
```
from now on you remember yourself we need three `*` surrounding commands!
